commit 6fc3a46295c8b5330e3611deb5edf97135e3dd24
Author: Dan Mick <dmick@redhat.com>
Date:   Tue Apr 11 16:27:33 2017 -0700

    mgr/PyModules handle_pyerror: don't add extra newlines to tb
    
    Signed-off-by: Dan Mick <dan.mick@redhat.com>

diff --git a/src/mgr/PyModules.cc b/src/mgr/PyModules.cc
index e77df011a8c..247e6b3fcd3 100644
--- a/src/mgr/PyModules.cc
+++ b/src/mgr/PyModules.cc
@@ -306,7 +306,7 @@ std::string handle_pyerror()
         object format_exception(traceback.attr("format_exception"));
         formatted_list = format_exception(hexc,hval,htb);
     }
-    formatted = str("\n").join(formatted_list);
+    formatted = str("").join(formatted_list);
     return extract<std::string>(formatted);
 }
 
